### PR TITLE
Job description page

### PR DESCRIPTION
**Pull request by 2016DigiTech**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>
for job description page 
Test URLs:
- Before: https://main--tlcgrp--hlxsites.hlx.page/
- After: https://job-description-page--tlcgrp--hlxsites.hlx.page/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "test:watch": "npm test -- --watch",
     "lint:js": "eslint .",
     "lint:css": "stylelint 'blocks/**/*.css' 'styles/*.css'",
-    "lint": "npm run lint:js && npm run lint:css"
+    "lint": " eslint . --ext js,jsx,ts,tsx --fix"
```
Comment (rofe) [inline package.json]: Please re-enable CSS linting.

**Review by rofe — CHANGES_REQUESTED** [review]
- What is the `.default-content-wrapper` file for?
- CSS does not follow any style guide (indentation, spaces between rules etc)
- Please re-enable CSS linting

**Comment by rofe** [comment]
> * What is the `.default-content-wrapper` file for?
image
Please remove this empty file from the PR.

**Comment by rofe** [comment]
Note: For linting to pass, you need to fix #67 as well.

File: package.json [hunk 2]
```diff
@@ -23,16 +23,16 @@
...
     "chai": "4.3.6",
     "eslint": "8.21.0",
     "eslint-config-airbnb-base": "15.0.0",
     "eslint-plugin-import": "2.26.0",
-    "@esm-bundle/chai": "4.3.4-fix.0",
-    "@web/test-runner": "0.13.31",
-    "@web/test-runner-commands": "0.6.3",
     "sinon": "14.0.0",
-    "stylelint": "14.9.1",
+    "stylelint": "^14.9.1",
```
Comment (rofe) [inline package.json]:
```suggestion
    "stylelint": "14.9.1",
```

File: package.json [hunk 3]
```diff
@@ -23,16 +23,16 @@
...
     "stylelint-config-prettier": "9.0.3",
-    "stylelint-config-standard": "25.0.0"
+    "stylelint-config-standard": "^25.0.0"
```
Comment (rofe) [inline package.json]:
```suggestion
    "stylelint-config-standard": "25.0.0"
```

**Review by rofe — CHANGES_REQUESTED** [review]
Don't use `^` prefixes for stable checkouts.

**Comment by 2016DigiTech** [comment]
Please review

**Review by 2016DigiTech — COMMENTED** [review]
Please review

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "javascript.preferences.quoteStyle": "single",
+  "typescript.preferences.quoteStyle": "single"
+}
```
Comment (rofe) [inline .vscode/settings.json:6]: I would not store editor settings with the project. Use your own local settings instead.

File: blocks/cards/cards.css [hunk 5]
```diff
@@ -303,32 +303,39 @@ main .section.cards-container .cards-wrapper {
...
-    left:0px;
-    right:0px;
+    left:0;
+    right:0;
     bottom:290px;
 }
+
 .meet-our-team-desktop-view .cards.review > ul > li .cards-card-body strong:first-of-type{
-    padding-top: 0px;
+    padding-top: 1px;
```
Comment (rofe) [inline blocks/cards/cards.css:309]: Is this change intentional, or should it be `0`?

File: blocks/cards/cards.css [hunk 6]
```diff
@@ -337,13 +344,15 @@ main .section.cards-container .cards-wrapper {
     font-style: italic;
     
 }
-.section.fragment-container.world-class-cutomer-loyality.cards-container p a {
 
+.section.fragment-container.world-class-cutomer-loyality.cards-container p a {
```
Comment (rofe) [inline blocks/cards/cards.css]: Note that there is a typo in the class name.

File: blocks/cards/cards.css [hunk 7]
```diff
@@ -354,60 +363,65 @@ main .section.cards-container .cards-wrapper {
...
+
 .world-class-cutomer-loyality .cards-card-body p em{
     color:white;
 }
+
 .world-class-cutomer-loyality .cards:not(.review) > ul > li:nth-child(2) a:any-link {
     color: white;
-}
-
+}}
```
Comment (rofe) [inline blocks/cards/cards.css]: This looks odd...

File: blocks/cards/cards.css [hunk 8]
```diff
@@ -354,60 +363,65 @@ main .section.cards-container .cards-wrapper {
...
 
+/* by azam here start code for mobile */
```
Comment (rofe) [inline blocks/cards/cards.css]: As illustrated in the [boilerplate](https://github.com/adobe/helix-project-boilerplate/), mobile should be the default, with tablet and desktop as exceptions with `@media (min-width:...)` queries.

File: blocks/cards/cards.css [hunk 9]
```diff
@@ -354,60 +363,65 @@ main .section.cards-container .cards-wrapper {
...
     main .section.cards-container .default-content-wrapper h3 {
         font-size:28px;
     }
+
     main .section.cards-container .default-content-wrapper em {
         font-size: 14px;
         font-style: normal;
     }
+
 /* try */
```
Comment (rofe) [inline blocks/cards/cards.css]: Try what?

File: blocks/cards/cards.css [hunk 10]
```diff
@@ -443,104 +459,114 @@ main .section.cards-container .cards-wrapper {
...
 .history-vision-section a:any-link a{
     color:white;
 }
+
 .history-vision-section h4#view-our-privacy-policy- a {
     color: white;
     font-size: 14px;
 }
+
 h3#we-follow-the-highest-standards-of-data-governance-and-are-compliant-with-all-prevailing-laws{
```
Comment (rofe) [inline blocks/cards/cards.css]: Do not use content-specific styling, use semantics (e.g. `.cards h3`) instead.

File: blocks/cards/cards.css [hunk 11]
```diff
@@ -443,104 +459,114 @@ main .section.cards-container .cards-wrapper {
...
     font-size:21px;
 }
+
 h4#tlc-compliances-and-statutory-audits-are-managed-by-a-big-4-global-accounting-firm-we-follow-the-highest-levels-of-information-security-with-an-iso-27001-certification-we-are-responsible-conscious-and-have-goals-of-net-zero-carbon-emissions{
```
Comment (rofe) [inline blocks/cards/cards.css]: Do not use content-specific styling, use semantics (e.g. `.cards h4`) instead.

File: blocks/cards/cards.css [hunk 12]
```diff
@@ -443,104 +459,114 @@ main .section.cards-container .cards-wrapper {
...
+    margin-top:0;
 }
 
-
-}
\ No newline at end of file
+.cards.fifty-fifty > ul > li .cards-card-body {
+    padding-top: 50px;
+    padding-bottom: 0;
+}}
```
Comment (rofe) [inline blocks/cards/cards.css:564]: Is this correct?

File: blocks/columns/columns.css [hunk 13]
```diff
@@ -286,43 +298,53 @@ h2#our-capabilities {
...
     main .section .default-content-wrapper em {
         font-size: 14px;
     }
+
     main .section .default-content-wrapper strong {
         font-size: 14px;
     }
+
     h3#product-manager, h3#data-analyst, h3#ionic-developer,
     h3#manager---influencer-marketing {
```
Comment (rofe) [inline blocks/columns/columns.css]: Do not use content-specific styling, use semantics (e.g. `.columns h3`) instead.

File: blocks/contact-center/contact-center.css [hunk 14]
```diff
@@ -41,6 +41,7 @@ main .contact-center img {
   main .contact-center h2 {
     font-size: 21px;
 }
```
Comment (rofe) [inline blocks/contact-center/contact-center.css:43]: Indentation problem

File: blocks/form/form.css [hunk 15]
```diff
@@ -1,3 +1,163 @@
+
+main .form {
+    padding: var(--spacing-m);
+  }
+  
+main .field-wrapper .required {
+    text-transform: capitalize;
+}
+  
+  main .form h2 {
```
Comment (rofe) [inline blocks/form/form.css]: Indentation problem

File: blocks/form/form.css [hunk 16]
```diff
@@ -1,3 +1,163 @@
...
+    text-transform: uppercase;
+    box-sizing: border-box;
+    font-weight: 800;
+  }
+  
+  /* main .form label.required::after {
+    content: "*";
+    color: red;
+    padding-left: 5px;
+  } */
```
Comment (rofe) [inline blocks/form/form.css]: Remove if not needed?

File: blocks/form/form.css [hunk 17]
```diff
@@ -1,3 +1,163 @@
...
+  border-bottom:1px solid rgb(193 193 195);
+  outline:none;
+  padding-left:0;
+}
+
+input[type='text-field']::placeholder {
+  font-size:20px;
+}
+
+textarea#yourMessage{
```
Comment (rofe) [inline blocks/form/form.css]: `#yourMessage` is not needed

File: blocks/fragment/fragment.css [hunk 18]
```diff
@@ -1 +0,0 @@
-/* nothing here */
```
Comment (rofe) [inline blocks/fragment/fragment.css:1]: Isn't stylelint complaining about empty CSS files?

File: package.json [hunk 19]
```diff
@@ -7,8 +7,7 @@
     "test": "wtr \"./test/**/*.test.js\" --node-resolve --port=2000 --coverage",
     "test:watch": "npm test -- --watch",
     "lint:js": "eslint .",
-    "lint:css": "stylelint 'blocks/**/*.css' 'styles/*.css'",
-    "lint": " eslint . --ext js,jsx,ts,tsx --fix"
+    "lint": "eslint . --ext js,jsx,ts,tsx --fix"
```
Comment (rofe) [inline package.json]: This should be revisited with #67 ...

File: blocks/header/header.css [hunk 20]
```diff
@@ -304,25 +304,28 @@ header nav a.active {
...
   background: #5a5a5a;
 }
+
 .nav-brand p {
   position: relative;
   left: 100%;
   display: inline-block;
-}
-}
+}}
```
Comment (rofe) [inline blocks/header/header.css]: Indentation problem

File: blocks/hero/hero.css [hunk 21]
```diff
@@ -82,27 +82,29 @@ main .hero h6 {
...
         position: absolute;
         bottom: 0;
         left: 60px;
         color: white;
         font-size: 0.7em;
         display: none;
-    }
-}
\ No newline at end of file
+    }}
```
Comment (rofe) [inline blocks/hero/hero.css:110]: Indentation problem

File: blocks/subnav/subnav.css [hunk 22]
```diff
@@ -38,5 +39,4 @@ main .subnav a:any-link {
     main .subnav p:hover {
         background:#484747;
         color: white;
-    }
-}
+    }}
```
Comment (rofe) [inline blocks/subnav/subnav.css:42]: Indentation problem

File: blocks/z-pattern/z-pattern.css [hunk 23]
```diff
@@ -76,6 +77,7 @@
 
 .z-pattern .z-row-even:last-of-type > div:last-of-type {
     text-align: center;
+
     /* z-index: 0; */
```
Comment (rofe) [inline blocks/z-pattern/z-pattern.css]: Remove empty lines and unused code.
```suggestion
```

File: scripts/delayed.js [hunk 24]
```diff
@@ -18,11 +18,6 @@ function loadScript(url, type, callback) {
   }
   return script;
 }
-
-window.targetGlobalSettings = {
-  bodyHidingEnabled: false,
```
Comment (rofe) [inline scripts/delayed.js:23]: Why did you remove this again? This was causing a white flash after a few seconds when Target kicks in...

**Review by rofe — CHANGES_REQUESTED** [review]

File: scripts/delayed.js (hunk 24 above)
Comment (2016DigiTech) [reply scripts/delayed.js:23]: I have not deleted it.

Comment (2016DigiTech) [reply scripts/delayed.js:23]: how is this getting deleted

File: blocks/z-pattern/z-pattern.css (hunk 23 above)
Comment (2016DigiTech) [reply blocks/z-pattern/z-pattern.css]: empty lines removed

File: blocks/cards/cards.css (hunk 9 above)
Comment (2016DigiTech) [reply blocks/cards/cards.css]: removed

File: blocks/cards/cards.css (hunk 11 above)
Comment (2016DigiTech) [reply blocks/cards/cards.css]: Used semantic tag

.cards h4{}

File: blocks/cards/cards.css (hunk 12 above)
Comment (2016DigiTech) [reply blocks/cards/cards.css:564]: yes

File: blocks/columns/columns.css (hunk 13 above)
Comment (2016DigiTech) [reply blocks/columns/columns.css]: Add semantics tag

File: blocks/form/form.css (hunk 15 above)
Comment (2016DigiTech) [reply blocks/form/form.css]: resolve

File: blocks/form/form.css (hunk 16 above)
Comment (2016DigiTech) [reply blocks/form/form.css]: Removed

File: blocks/form/form.css (hunk 17 above)
Comment (2016DigiTech) [reply blocks/form/form.css]: Removed

File: package.json (hunk 19 above)
Comment (2016DigiTech) [reply package.json]: Sure

File: blocks/header/header.css (hunk 20 above)
Comment (2016DigiTech) [reply blocks/header/header.css]: Resolved

File: blocks/subnav/subnav.css (hunk 22 above)
Comment (2016DigiTech) [reply blocks/subnav/subnav.css:42]: Resolved

**Comment by 2016DigiTech** [comment]
Please review
